### ISSUE TITLE
How I can  setup AUTO1111 formatted LoRA alpha ?

### ISSUE THREAD

**Issue by stevensu1977** [huggingface/diffusers#3743]
from source code I found network_alpha , but how I can setup network_alpha ?

# Convert kohya-ss Style LoRA attn procs to diffusers attn procs
network_alpha = None
        if all((k.startswith("lora_te_") or k.startswith("lora_unet_")) for k in state_dict.keys()):
            state_dict, network_alpha = self._convert_kohya_lora_to_diffusers(state_dict)

**Comment by patrickvonplaten** [comment]
Hey @stevensu1977,

Sorry I don't understand the question - could you clarify what is meant here exactly?

**Comment by stevensu1977** [comment]
sorry , I change my question ,  I try load multiple lora (invoke load_lora_weight  x times) , but I think it's not work.

**Comment by stevensu1977** [comment]
@patrickvonplaten  in auto1111/sd-weui , use "alpha" parameter setup LoRA model effect ,   in diffusers same parameter is  "lora_scale" ?

**Comment by patrickvonplaten** [comment]
I'm not 100% sure. Note that `"alpha"` and `scale` is different. Often `scale = alpha / r` with r being the lora embedding dimension. Also cc @pacman100 - do you know where "lora" alpha comes from exactly?

**Comment by pacman100** [comment]
Hello, as @patrickvonplaten mentioned `scale = alpha / r`. `alpha` is a hyperparam that user provides generally.

**Comment by pacman100** [comment]
It is used for scaling as mentioned above, see below:

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/baf6946e06249c5af9851c60171692c44ef633e0/extensions-builtin/Lora/lora.py#LL286C46-L286C86

>

**Comment by alexblattner** [comment]
use this: 
[loraLoader.txt](https://github.com/huggingface/diffusers/files/11879534/loraLoader.txt)

thank me later. Works perfectly.

**Comment by hckj588ku** [comment]
> use this: [loraLoader.txt](https://github.com/huggingface/diffusers/files/11879534/loraLoader.txt)
> 
> thank me later. Works perfectly.

Hi,I use this with error:expected scalar type Half but found Floatdo you know why?

**Comment by alexblattner** [comment]
@hckj588ku yes actually, I happened to have the same problem. I should have fixed it though. let me look into it. Can I see your code and lora?

**Comment by hckj588ku** [comment]
> @hckj588ku yes actually, I happened to have the same problem. I should have fixed it though. let me look into it. Can I see your code and lora?

https://civitai.com/models/12597/moxin  this is lora address

and my code is 

` 

            pipeline = StableDiffusionPipeline.from_pretrained(Config.model_path, torch_dtype=torch.float16)
            callback = sample_callback
            pipeline.to(DEVICE)
            if 'runFX' not in Config.model_path and 'rmada' not in Config.model_path and Config.lora_path:
                kohya_lora_loader.install_lora_hook(pipeline)
                pipeline.apply_lora(Config.lora_path, 1.0)
                use_lora = True
            # text_encoder = CLIPTextModel.from_pretrained(Config.model_path,
            #                                              subfolder="text_encoder", num_hidden_layers=11,
            #                                              torch_dtype=torch.float16).to(DEVICE)
            # pipeline.text_encoder = text_encoder
            pipeline.scheduler = DDIMScheduler.from_config(pipeline.scheduler.config)
            outputs = pipeline([prompts],
                               negative_prompt=[Config.negative_prompt],
                               num_images_per_prompt=len(seeds),
                               generator=generator,
                               width=Config.width,
                               height=Config.height,
                               guidance_scale=Config.scale,
                               num_inference_steps=Config.steps,
                               callback=callback,
                               callback_steps=8,
                               ).images

`

**Comment by alexblattner** [comment]
[loraLoader.txt](https://github.com/huggingface/diffusers/files/12105238/loraLoader.txt)
this works

**Comment by hckj588ku** [comment]
> [loraLoader.txt](https://github.com/huggingface/diffusers/files/12105238/loraLoader.txt) this works

I works,thanks a lot
